### PR TITLE
Do not raise an exception when conditional field is not visible 

### PR DESCRIPTION
**Pull request by alexdashkov**
current user type

## Review

This PR closes #

* [x] Tests
* [x] `CHANGELOG.rst` Updated
* [x] Delete your branch

### CODE REVIEW

File: demo/tests/test_conditions.py [hunk 1]
```diff
@@ -187,6 +193,15 @@ def test_readonly_field_not_displayed(self):
         self.assertTrue(form.is_valid(), form.errors)
         self.assertEqual(form.cleaned_data, {'checkbox': True})
 
+    def test_condition_field_doesnt_exist(self):
```
Comment (wo0dyn) [inline demo/tests/test_conditions.py]: Add description about the test:
* `wrong-conditions.json` viewed as an Employee (that's why, some fields are missing, b/c employee doesn't have the right to see them)
* `self.assertTrue()` is here to show we don't raise an exception anymore.

File: formidable/forms/conditions.py [hunk 2]
```diff
@@ -39,8 +39,11 @@ def gen_from_schema(self, fields, conditions_schema):
 
             # cast values to field's type
             def convert_values(field_id, values):
-                field = fields[field_id]
-                return [field.to_python(value) for value in values]
+                try:
+                    field = fields[field_id]
+                    return [field.to_python(value) for value in values]
+                except KeyError:
+                    return []
```
Comment (wo0dyn) [inline formidable/forms/conditions.py]:
```py
else:
    return [field.to_python(value) for value in values]
```

File: demo/tests/test_conditions.py [hunk 3]
```diff
@@ -187,6 +193,15 @@ def test_readonly_field_not_displayed(self):
...
+        schema = json.load(open(
+            os.path.join(
+                TESTS_DIR, 'fixtures', 'wrong-conditions.json'
+            )
+        ))
+        form_class = get_dynamic_form_class_from_schema(schema)
+        self.assertTrue(issubclass(form_class, BaseDynamicForm))
```
Comment (wo0dyn) [inline demo/tests/test_conditions.py]:
```py
try:
  foo()
except Exception:
  self.fail()
else:
  self.asssertTrue(True)
```

**Comment by alexdashkov** [comment]
@wo0dyn Thank you for your review. You could check the changes.

**Pull request merged**